Remove confusing text; fix code example (#3393)
quiltdata/quilt: docs/walkthrough/working-with-elasticsearch.md

<!-- markdownlint-disable -->

Each Quilt stack includes an Elasticsearch cluster that indexes

> objects and packages. To write specialized queries against your
> data stored in Amazon S3 buckets you may wish to connect directly
> to your Quilt Elasticsearch cluster.

Reviewer: ```suggestion
objects and packages as documents. You can connect to your Elasticsearch
domain to query these documents.

> If your Quilt stack uses private endpoints for Elasticsearch you will need to
> connect to the cluster from a machine in the same VPC as the cluster.
```

Each Amazon S3 bucket connected to Quilt implies two Elasticsearch index aliases:
1. `YOUR_BUCKET_NAME`: Contains one document per object in the bucket.
2. `YOUR_BUCKET_NAME_packages`: Contains one document per package revision in the bucket.

> > Quilt uses Amazon Elasticsearch version 6.7

Reviewer: ```suggestion
> Quilt uses Amazon Elasticsearch version 6.7.
```

<!-- markdownlint-disable -->

> The [Quilt web catalog

Reviewer: ```suggestion
Each Quilt stack includes an Elasticsearch cluster that indexes objects and packages.
```

search](../walkthrough/working-with-the-catalog#search) is powered

> by Elasticsearch. To write specialized queries against your data

Reviewer: Eliminate

stored in Amazon S3 buckets you may wish to connect directly to

> your Quilt Elasticsearch cluster.

Reviewer: Eliminate

> > Note that Quilt is currently pinned to Elasticsearch 6.7

Reviewer: ```suggestion
> Quilt uses Elasticsearch 6.7
```

> Each Amazon S3 bucket connected to Quilt has two Elasticsearch indexes

Reviewer: ```suggestion
Each Amazon S3 bucket connected to Quilt implies two Elasticsearch index aliases:
```

> with the following aliases:

Reviewer: ```suggestion
```

1. `YOUR_BUCKET_NAME`: Contains one document per object in the bucket.
2. `YOUR_BUCKET_NAME_packages`: Contains one document per package revision in the bucket.

> ## Querying Elasticsearch with Python

Reviewer: ```suggestion
## Query Elasticsearch with Python
```

> You can use the [`Elasticsearch`
> API](https://www.elastic.co/guide/en/elasticsearch/reference/6.7/) to
> query your cluster.

Reviewer: This is a PyPI module, should be lower case and here is pinned to the highest version less than or
equal to 6.7 for compatibility.
```suggestion
You can use [`elasticsearch 6.3.1`](https://elasticsearch-py.readthedocs.io/en/6.3.1/) as follows.
```

    }
}

elastic.search(
    index="*_packages", # search all package indexes in this stack
    body=rbody,

>     _source=['*'], # return all fields in your documents

Reviewer: ```suggestion
    _source=['*'], # return all document fields
```
